app/controllers/sites_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
 
     respond_to do |format|
       if @site.save
-        format.html { redirect_to sites_path, notice: 'Site was successfully created.' }
+        format.html { redirect_to sites_path, notice: t(".notice") }
```

Comment:
```suggestion
        format.html { redirect_to sites_path, notice: t(".flashes.created") }
```

@B-Rass what do you think about? Any other suggestion?

Reply:
Whad do I think about what ?

Reply:
my code suggestion, to be more precise on flash message key in controllers. Maybe there is a better key syntax

Reply:
I can't find a situation where more precision can be a bad decision here

Reply:
sure